pages/api/cities/[city].js

```diff
@@ -26,40 +26,52 @@ handler.get(async (req, res) => {
     state,
     population,
     CO2,
-    latitude,
-    longitude,
     waterpH,
     totalDissolvedSolids,
     specificConductance,
   } = doc;
 
-  // TODO: get air quality
   const response = await fetch(
+    "https://api.opencagedata.com/geocode/v1/json?q=" +
+      name +
+      "," +
+      state +
+      ",United%20States" +
+      "&key=" +
+      process.env.LONGLAT_KEY
+  );
+
+  console.log(response);
+
```

Comment:
maybe we could do `const latitude = response.results[0].bounds.northeast.lat;` here? That way we can just reference "latitude" further down instead of needing to write the whole thing out.

also, is there a value for northeast and for southwest? maybe we could average them to get more of a central location?

Reply:
Sounds good to me. Will fix it right now